FoodDeliveryUI/FoodDeliveryUI/FoodDeliveryUI.Shared/MainPage.xaml

```diff
@@ -0,0 +1,80 @@
+﻿<Page
+    x:Class="FoodDeliveryUI.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:FoodDeliveryUI" 
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"   
+    Background="#09153c">
+    <!-- Page layout structure-->
+    <Grid Padding="30,50,30,0" ColumnSpacing="20" RowDefinitions="Auto,Auto,Auto,Auto,Auto,Auto,Auto" ColumnDefinitions="*,*,Auto">

```

Comment:
also consider wrapping the entire root Grid inside a ScrollViewer, so the view experience on smaller device isnt limited.